crates/rpc/rpc/src/debug.rs

```diff
@@ -733,12 +733,20 @@ where
 
                 let mut witness_record = ExecutionWitnessRecord::default();
 
-                let _ = block_executor
+                let output = block_executor
                     .execute_with_state_closure(&block, |statedb: &State<_>| {
                         witness_record.record_executed_state(statedb);
                     })
                     .map_err(|err| EthApiError::Internal(err.into()))?;
 
+                reth_ethereum_consensus::validate_block_post_execution(
```

Comment:
This is similar to the [check here](https://github.com/paradigmxyz/reth/blob/ba84eeaccd1c9afc64bb18d5957cefe69aec8212/crates/stateless/src/validation.rs#L234-L235), which is how I realized in L1 proving that the witness was wrong.